lib/blockchain/hardwareWallet.js

```diff
@@ -0,0 +1,305 @@
+const assert = require('bsert');
```

Comment:
Is this really "blockchain business logic"? It doesn't really touch the blockchain. feels more like wallet or tx business logic.

Reply:
I'd consider wallet and transaction logic to be blockchain specific logic

Reply:
My feeling is that that's too general. From this view, pretty much all of our business logic is blockchain logic, including working with accounts, wallets, etc. Considering the way bcoin is organized for example, `blockchain` is very specific to things on the chain, i.e. blocks: https://github.com/bcoin-org/bcoin/tree/master/lib/blockchain

Reply:
Do you have a suggestion as to a better name?

Reply:
Maybe something like utilities? I guess it depends what else would go in a folder like this. Since we envision eventually moving this into bpanel-utils, what would be left here? Most of the top level organization for our front end directories is done by function (actions, componenents, reducers, etc). Keeping with that this feels like a utility or helper.

Reply:
Updated